_templates/_search_origin.html.erb

```diff
@@ -1,12 +1,15 @@
+<div id="hc-search">
+  <input id="hc-search-input" type="text">
+  <button id="hc-search-btn">Search</button>
+</div>
+
+<div id="hc-search-modal">
+  <div id="hc-modal-content">
+    <span id="hc-modal-close">&times;</span>
+    <div id="hc-search-results"></div>
+  </div>
+</div>
+
 <script>
-  (function() {
-    var cx = '013769182564158614814:vdyz7waqya4';
-    var gcse = document.createElement('script');
-    gcse.type = 'text/javascript';
-    gcse.async = true;
-    gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') + '//cse.google.com/cse.js?cx=' + cx;
-    var s = document.getElementsByTagName('script')[0];
-    s.parentNode.insertBefore(gcse, s);
-  })();
+  hcSearchCategory("docs_origin");
 </script>
-<gcse:search defaultToRefinement="<%= version %>"></gcse:search>
```

Comment:
See above comment about use of refinements. If a user is searching, for instance, in the 'https://docs.openshift.com/container-platform/3.4/*' part of the website, we want the search results to be limited to other pages under the same URL path.